geps/gep-713/metadata.yaml

```
@@ -2,21 +2,22 @@ apiVersion: internal.gateway.networking.k8s.io/v1alpha1
 kind: GEPDetails
 number: 713
 name: Metaresources and Policy Attachment
-status: Memorandum
+status: Standard
```

This is a follow up to my comment above TODO:

```suggestion
status: Memorandum
```

---

Done